sdk/search/search-documents/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@azure/search-documents",
-  "version": "11.2.0-beta.1",
+  "version": "11.2.0-beta.2",
```

Comment:
We have automation to make changes like these when we release a package. See #14758

Any reason why we are not using the automated PRs?

Reply:
Missed that. I have closed it now.